app/lib/server/startup/settings.js

```diff
@@ -1022,6 +1022,13 @@ settings.addGroup('General', function() {
 			i18nDescription: 'Notifications_Max_Room_Members_Description',
 		});
 	});
+	this.section('User_Action_Indicators', function() {
+		this.add('Use_New_Action_Indicator', true, {
```

Comment:
I'd use the other way around here. Something like `Fire_Old_Typing_Event` and would mention it's a compatibility feature in the description

Reply:
Of course it would keep firing the new event but would fire the old as well

Reply:
We will fire only the old typing action (if the option is allowed). Handle function is capable of handling them both. So no need to fire twice.